go/pkg/trainingjober/trainingjober.go

```diff
@@ -0,0 +1,62 @@
+// trainingjober is a package for managing a TrainingJob
+package trainingjober
+
+import (
+	"github.com/PaddlePaddle/cloud/go/pkg/apis/paddlepaddle/v1alpha1"
+	trainingJobClient "github.com/PaddlePaddle/cloud/go/pkg/client/clientset/versioned"
+	"k8s.io/client-go/kubernetes"
+	"time"
+)
+
+const (
+	retryTimes            = 5
+	convertedTimerTicker  = 10 * time.Second
+	confirmResourceTicker = 5 * time.Second
+)
+
+type trainingJobEventType string
+
+const (
+	trainingJobEventDelete trainingJobEventType = "Delete"
+	trainingJobEventModify trainingJobEventType = "Modify"
+)
+
+type trainingJobEvent struct {
+	// pet is the TrainingJobEventType of TrainingJob
+	pet trainingJobEventType
+	// The job transfer the information fo job
```

Comment:
`The job transfer the information fo job `, Do you mean `job is the reference to TrainingJob` ?